drivers/Makefile.am

```diff
@@ -164,6 +171,11 @@ upscode2_LDADD = $(LDADD) -lm
 victronups_SOURCES = victronups.c
 riello_ser_SOURCES = riello.c riello_ser.c
 riello_ser_LDADD = $(LDADD) -lm
+#gpio_SOURCES = generic_gpio_common.c generic_gpio_libgpiod.c
+generic_gpio_libgpiod_SOURCES = generic_gpio_common.c generic_gpio_libgpiod.c
+#gpio_CFLAGS = $(AM_CFLAGS)
+#gpio_LDADD = $(LDADD_DRIVERS) -lgpiod
+generic_gpio_libgpiod_LDADD = $(LDADD_DRIVERS) -lgpiod
```

Comment:
I'm a bit split on whether explicit `-lgpiod` belongs here. Consistently, a `LIBGPIOD_LIBS` as detected by `configure.ac` and `m4` snippet, would be better (especially if some more custom flags come later or on certain platforms).

Also, I think this block belongs lower in the file, among "non-serial drivers" - e.g. near I2C as the added lines neighbor elsewhere.